src/app/src/util/constants.facilitiesMap.js

```diff
@@ -7,29 +7,6 @@ export const initialCenter = Object.freeze({
     lng: 5,
 });
 
-export const initialZoom = 1;
+export const initialZoom = 3;
```

Comment:
Consider 2 as an initial zoom rather than 3. It shows more of the world.

#### Zoom 3

![Screen Shot 2019-07-30 at 15 58 59](https://user-images.githubusercontent.com/17363/62172020-d312f280-b2e5-11e9-8fdf-aba41ba13a1f.png)

#### Zoom 2

![Screen Shot 2019-07-30 at 15 59 14](https://user-images.githubusercontent.com/17363/62172048-e2923b80-b2e5-11e9-911d-e5688370be15.png)

Reply:
Changed this in fcf2a55b16753cc32160ac0e74a71ba06eb2277e